lib/integration/nation_builder.rb

```diff
@@ -12,9 +12,9 @@ class Integration::NationBuilder
   ALLOWED_PARAMS_EVENT = [:slug, :status, :name, :start_time, :end_time,
     autoresponse: [:broadcaster_id, :subject, :body]]
   ALLOWED_PARAMS_PERSON = [:birthdate, :do_not_call, :first_name, :last_name, :email, :email_opt_in, :employer, :is_volunteer, :mobile_opt_in,
-    :mobile, :occupation, :phone, :recruiter_id, :sex, :tags, :request_ip, :skills, :rootstrikers_subscription, :uuid,
+    :mobile, :occupation, :phone, :recruiter_id, :sex, :request_ip, :skills, :rootstrikers_subscription, :uuid,
```

Comment:
Line is too long. [111/80]<br>Align the elements of an array literal if they span more than one line.